examples/validators/policy-controller-validate/invalid-configmap/README.md

```diff
@@ -0,0 +1,73 @@
+# policy-controller-validate: invalid configmap
```

Comment:
Current OSS image is called `gatekeeper-validate`, there's a non-OSS image called `policy-controller-validate` as well, but it's not this.

https://googlecontainertools.github.io/kpt/guides/consumer/function/catalog/validators/